global_costmap:
   global_frame: map1
   robot_base_frame: base_footprint1
   update_frequency: 5.0
   publish_frequency: 1.0
   static_map: true
   rolling_window: false
   width: 15.0
   height: 6.5
   resolution: 0.05
   transform_tolerance: 0.5
